src/mutualinfo.jl

```diff
@@ -0,0 +1,40 @@
+# Mutual Information
+
+function _mutualinfo(A::AbstractMatrix{<:Integer}, normed::Bool)
+    N = sum(A)
+    (N == 0.0) && return 0.0
+
+    rows = sum(A, dims=2)
+    cols = sum(A, dims=1)
+    entA = entropy(A)
+    entArows = entropy(rows)
+    entAcols = entropy(cols)
+
+    hck = (entA - entAcols)/N
+    hc = entArows/N + log(N)
+    hk = entAcols/N + log(N)
+
+    mi = hc - hck
+    return if normed
+        2*mi/(hc+hk)
+    else
+        mi
+    end
+end
+
+"""
+    mutualinfo(a, b; normed=true) -> Float64
+
+Compute the *mutual information* between the two clusterings of the same
+data points.
+
+`a` and `b` can be either [`ClusteringResult`](@ref) instances or
+assignments vectors (`AbstractVector{<:Integer}`).
+
+If `normed` paremeter is `true`, the normalized mutual information (sum) is returned.
```

Comment:
```suggestion
If `normed` parameter is `true`, the normalized mutual information (sum) is returned.
```

Reply:
I'm not sure I 100% understand "normalized mutual information (sum)" mean. What is "sum" in this context?